src/main/java/seedu/address/model/reminder/Reminder.java

```diff
@@ -0,0 +1,132 @@
+package seedu.address.model.reminder;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+import java.util.Properties;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+import seedu.address.model.person.Person;
+import seedu.address.model.person.User;
+import seedu.address.model.transaction.Loan;
+
+/**
+ * Represents a reminder, which reminds people on their unpaid debts through emails.
+ */
+public class Reminder {
+    private User sender;
+    private Person receiver;
+    private MimeMessage message;
+
+    private final String username = "noreply.loansharkie";
+    private final String password = "cs2103t-w12-3";
```

Comment:
might be clearer to put `emailUsername` and `emailPassword`, so that we know this username and password is for email.